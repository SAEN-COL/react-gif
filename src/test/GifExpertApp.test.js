import React from "react";
import '@testing-library/jest-dom';
import { shallow } from "enzyme";
import GifExpertApp from '../GifExpertApp.js';

describe('Se crea correctamente el componente <GifExpertApp />', () => {
  test('DeberÃ­a generar correctamente el componente', () => {
    const wrapper = shallow(<GifExpertApp />);
    expect( wrapper ).toMatchSnapshot();
  });

  
});
