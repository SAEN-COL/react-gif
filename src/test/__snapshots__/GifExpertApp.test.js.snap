// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Se crea correctamente el componente <GifExpertApp /> Deber√≠a generar correctamente el componente 1`] = `
<Fragment>
  <h2>
    GifExpertApp
  </h2>
  <AddCategory
    setCategories={[Function]}
  />
  <hr />
  <ol>
    <GifGrid
      category="Chanfle"
      key="Chanfle"
    />
  </ol>
</Fragment>
`;
